client/src/components/QA/QAAddAnswerModal.jsx

```diff
@@ -21,28 +13,35 @@ const QAAddAnswerModal = ({ question, productName }) => {
 
   return (
     <div data-testid="qa-div-card-questions">
-      <button onClick={() => setModalIsOpenAdd(true)} className="qa-add-answer" href=""><b>[Add Answer]</b></button>
-      <Modal isOpen={modalIsOpenAdd} onRequestClose={() => setModalIsOpenAdd(false)}>
-        <h2>Submit Your Answer</h2>
-        <h3>{productName.name}: {question.question_body} </h3>
-        <form className="add-answer-form">
-          <label className="add-form-answer">Answer* </label>
-            <textarea rows="10" maxLength="1000" required onChange={() => setAnswer(event.target.value)}/><p></p>
-          <label className="add-form-username">Nickname*</label>
-            <input maxLength="60" placeholder="Example: jack543!" required onChange={() => setNickname(event.target.value)}/><p className="qa-form-small">For privacy reasons, do not use your full name or email address</p>
-          <label className="add-form-email">Email*</label>
-            <input maxLength="60" placeholder="Example: jack@email.com" required onChange={() => setEmail(event.target.value)}/><p className="qa-form-small">For authentication reasons, you will not be emailed</p>
-          <button className="qa-add-form-submit" onClick={() => {
-            event.preventDefault();
-            setModalIsOpenAdd(false);
-            setModalInfo({
-              answer: answer,
-              nickname: nickname,
-              email: email
-            });
-          }}>Submit</button>
-        </form>
-      </Modal>
+      <button id="qa-modal-button" className="qa-add" onClick={() => {
```

Comment:
I'm curious why you're creating your own modal instead of using the one Sam built. This would make it easier to make a QAAddAnswer component and just pass it into the Modal wrapper. This would also let you refactor out a lot of the display block/none logic you have here, making it more tidy to read.